app/javascript/application.js

```diff
@@ -0,0 +1,21 @@
+// Entry point for the build script in your package.json
+// import "@hotwired/turbo-rails"
+// import "./controllers"
```

Comment:
- [ ] It's considered good practice to keep your code clean of any commented (unused) code in your codebase. Please remove these two commented lines 2,3.